MineSweeper.py

```diff
@@ -0,0 +1,256 @@
+import Tkinter as tk
+import random
+
+import sys
+sys.path.append('~/Library/Python/2.7/site-packages')
+
+from drawingpanel import *
+
+BOARD_HEIGHT = 300
+BOARD_WIDTH = 300
+BOX_SIZE = 20
+
+NUMBER_MINES = ((BOARD_WIDTH / BOX_SIZE - 1)**2)//10 
```

Comment:
Integer division. This is just to make sure that the number of mines is an integer because we will iterate over it